themes/delphi/layouts/_default/about.html

```diff
@@ -1,23 +1,13 @@
 {{ define "main" }}
 
-<div class="about-mission uk-cover-container">
-    <img class="about-mission-img" src="{{ "images/mission.jpg" | relURL }}" alt="Mission image" uk-cover />
-    <div class="about-mission-mission">
-        <div class="uk-container content-grid">
-            <div class="grid-2-10">
-                Our Mission
-            </div>
-        </div>
-    </div>
-    <div class="uk-container content-grid">
-        <h1 class="grid-2-10 about-mission-text">
-            {{.Site.Params.mission}}
-        </h1>
-    </div>
+<div class="about-mission">
 </div>
-
 <div class="uk-container content-grid">
     <div class="grid-2-10 about-description">
```

Comment:
doesn't seem to be fully centered in the grid:

![image](https://user-images.githubusercontent.com/4129778/100268271-bfe85500-2f22-11eb-8bbb-b225794444f1.png)

Reply:
This seems to have been an conscious design decision from Raph.  Let's defer.